src/main/java/com/amazonaws/encryptionsdk/AwsCrypto.java

```diff
@@ -281,7 +281,12 @@ public long estimateCiphertextSize(
     /**
      * Calls {@link #encryptData(MasterKeyProvider, byte[], Map)} on the UTF-8 encoded bytes of
      * {@code plaintext} and base64 encodes the result.
+     * @deprecated Use the {@link #encryptData(MasterKeyProvider, byte[], Map)} and
+     * {@link #decryptData(MasterKeyProvider, byte[])} APIs instead. {@code encryptString} and {@code decryptString}
```

Comment:
I would break on the `.`

Reply:
@deprecated Use the {@link #encryptData(MasterKeyProvider, byte[], Map)} and
{@link #decryptData(MasterKeyProvider, byte[])} APIs instead. {@code encryptString} and {@code decryptString} 
work as expected if you use them together. However, to work with other language implementations of the AWS Encryption  SDK, you need to base64 decode the output of {@code encryptString} and base64-encode the input to 
{@code decryptString}. These deprecated APIs will be removed in the future.